### ISSUE TITLE
Audio fault on Lenovo 100E Gen 2 Intel Celelron N4020

### ISSUE THREAD

**Issue by BonzoFerret** [cb-linux/breath#305]
Audio fault

Lenovo 100E Gen 2 Intel Celelron N4020 4GB 32GB eMMC 11.6 Inch Chrome OS Chromebook

Running Ubuntu with the Mate desktop, using the sof-setup-audio and SOUNDCARD=rtk sof-setup-audio scripts.  It does give a sound when the login screen asks for a password, but in the desktop there is no sound when playing youtube videos and the sound icon in the panel is permanently muted and can't be changed.

Otherwise this is a fantastic and very useable implementation of Ubuntu.  Wel done and many thanks.

**Comment by BonzoFerret** [comment]
[alsa-info](http://alsa-project.org/db/?f=a0fbea17262493593a81465f66715ed9627710f9)

**Comment by MilkyDeveloper** [comment]
Run:
```
systemctl --user mask pipewire.socket
systemctl --user mask pipewire.service
```
and reboot. Thanks for the praise :D. Feel free to donate to [Breath](https://opencollective.com/breath).

**Comment by BonzoFerret** [comment]
Thanks for the quick reply.  Unfortunately this doesn't seem to have fixed things, unless I'm doing something wrong.

**Comment by BonzoFerret** [comment]
Aha!  Some progress.  I have found that, after booting, if I run:

pulseaudio --kill
pulseaudio --start

I now get working sound.  

So all I now need to know is how to make it work straight from booting.

**Comment by MilkyDeveloper** [comment]
Would you mind joining the [Discord](https://discord.gg/MjPKmWDr)? We could debug this issue faster there.

**Comment by BonzoFerret** [comment]
I'd love to, but it seems Discord thinks I've breached its terms of service before I even managed to get the account set up.

**Comment by BonzoFerret** [comment]
OK some feedback.
I rebuilt breath from scratch and re-burned my SD card using ubuntu and the mate desktop.  
Audio still did not work after running sof-setup-audio but running this extended the boot time considerably
I then realised that I hadn't run 

systemctl --user mask pipewire.socket
systemctl --user mask pipewire.service

I did that, and rebooted.  Still no audio.
I then ran sof-setup-audio again and rebooted - still the longer reboot time, but this time hurrah! I have audio.
Audio is at maximum volume after reboot.  I also note that, even though I turn it down, it's still at maximum volume after a subsequent reboot.
